public class ThrowExample {
    public static void main(String[] args) {
        int a = 10;
        int b = 0;

        try {
            if (b == 0) {
                throw new ArithmeticException("Division by zero is not allowed!");
            }
            int result = a / b;
            System.out.println("Result: " + result);
        } catch (ArithmeticException e) {
            System.out.println("Caught exception: " + e.getMessage());
        }
    }
}
